<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_gacylinder</class>
 <widget class="QDialog" name="dlg_gacylinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>471</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Volume Tracer</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pRunGa">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Basic Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Trace</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="m_pDetectHelices">
           <property name="text">
            <string>Helices                      </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="m_pDetectFilaments">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Filaments               </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>General Settings</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_pHelicesCountText_2">
         <property name="text">
          <string>Number of Traced Objects:   </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="m_pHelicesCount">
         <property name="toolTip">
          <string>The nomber of helices or filaments to be detected</string>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="textLabel3_3">
         <property name="text">
          <string>Expansion Threshold:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="m_pAcceptMoveRatio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>influences how 'far' the expansion should progresses</string>
         </property>
         <property name="text">
          <string>70</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Map</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="m_pVoxelWidthLabel">
         <property name="text">
          <string>Voxel Width:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="m_pVoxelWidth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>2.0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="m_pVoxelUnits">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="m_pRes">
         <property name="toolTip">
          <string>Resolution of the input map</string>
         </property>
         <property name="text">
          <string>8.0</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="m_pResUnit">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Anisotropic correction</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Resolution anisotropy factor (Z vs XY)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="m_pAni">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Anosotropic correction along the z axis</string>
         </property>
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Orientation-dependent 
attenuation parameter (Z vs XY)</string>
         </property>
         <property name="buddy">
          <cstring>label_11</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="m_pLambda">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Correction term; 1 - means no correction; </string>
         </property>
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Template Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>General Settings</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel2_6">
         <property name="text">
          <string>Distance Between Segments:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="textLabel2_4">
         <property name="text">
          <string>Radius:                                     </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="m_pExpansionTemplateRadius">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>1.00</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="m_pTemplateRadiusUnit">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="textLabel4_4">
         <property name="text">
          <string>Segments:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="m_pExpansionTemplateLength">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Search Template</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="textLabel2_5">
         <property name="text">
          <string>Radius:  </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="m_pSearchTemplateRadius">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>2.00</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="m_pSearchTemplateRadiusUnit">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="textLabel4_5">
         <property name="text">
          <string>Segments:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="m_pSearchTemplateLength">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="m_pSearchTemplateRadiusUnit_2">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Expansion Template</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_pDistanceBetweenRepeats">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>1.00</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="m_pShowSearchTemplate">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="m_pShowExpansionTemplate">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Advanced Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_pPopSizeLabel">
         <property name="text">
          <string>Population Size:                  </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_pPopSize">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_pGenLabel">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Generations:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="m_pMaxGen">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30000</number>
         </property>
         <property name="singleStep">
          <number>25</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_pThreadsLabel">
         <property name="text">
          <string>Threads:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="m_pMaxThread">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_pSyncGenLabel">
         <property name="text">
          <string>Syncronize at Gen:        </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="m_pSyncGen">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="m_pRegionSize">
         <property name="text">
          <string>Tabu Region Size:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="m_pTabuRegionSize">
         <property name="text">
          <string>6.0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="m_pTabuRegionSizeUnit">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="m_pCrawlingStepUnit">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="m_pCrawlingStep">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>1.4</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="textLabel2_2">
         <property name="text">
          <string>Expansion Step:                     </string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_pRunGa</tabstop>
  <tabstop>pushButton1</tabstop>
  <tabstop>tabWidget2</tabstop>
 </tabstops>
 <includes>
  <include location="local">sculptor_document.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>dlg_gacylinder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRunGa</sender>
   <signal>clicked()</signal>
   <receiver>dlg_gacylinder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
